app/src/main/java/com/example/playlistmakervmp/OnTrackClickListener.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.playlistmakervmp
+
+interface OnTrackClickListener {
```

Comment:
```suggestion
fun interface OnTrackClickListener {
```
Этот интерфейс можно сделать функциональным. Что это такое и какие преимущества даёт можешь прочитать в официальной документации котлин - https://kotlinlang.org/docs/fun-interfaces.html#sam-conversions